### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by yeunga**
Changes include my changes for story 2093 and Helena's changes.

### CODE REVIEW

File: caom2-search-server/src/main/resources/META-INF/resources/images/wmms.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (at88mph) [inline caom2-search-server/src/main/resources/META-INF/resources/images/wmms.svg]: This file should be in AdvancedSearch only.  It does not apply to this open source version.

File: caom2-search-server/src/main/resources/META-INF/resources/index.jsp [hunk 2]
```diff
@@ -19,15 +19,16 @@
...
 
-<c:import url="_page_header.html"/>
+<c:import url='<%= "_page_header.jsp?lang=" + requestHeaderLang %>'/>
 
 <body>
 
 <div class="container-fluid">
 
 <%-- MainContentStart --%>
+    <c:import url="../../../../../../../wwebui/AdvancedSearch/src/main/webapp/_topnav.jsp"/>
```
Comment (at88mph) [inline caom2-search-server/src/main/resources/META-INF/resources/index.jsp]: This should only exist in AdvancedSearch, not in this open source version.

File: caom2-search-server/src/main/resources/META-INF/resources/images/wmms.svg (hunk 1 above)
Comment (yeunga) [inline caom2-search-server/src/main/resources/META-INF/resources/images/wmms.svg]: Okay. Will move it. Thanks.

File: caom2-search-server/src/main/resources/META-INF/resources/index.jsp (hunk 2 above)
Comment (yeunga) [inline caom2-search-server/src/main/resources/META-INF/resources/index.jsp]: Removed this line.

**Pull request merged**